matchmaker/views/matchmaker_api.py

```diff
@@ -150,7 +152,7 @@ def _search_external_matches(nodes_to_query, patient_data):
                 try:
                     message = external_result.json().get('message')
                 except Exception:
-                    message = external_result.content
+                    message = external_result.content.decode('utf-8')
```

Comment:
`external_result.content` from the RESTful API is binary. So decoding is required.